features/netsocket/nsapi_dns.cpp

```diff
@@ -105,10 +105,18 @@ static nsapi_addr_t dns_servers[DNS_SERVERS_SIZE] = {
     {NSAPI_IPv4, {8, 8, 8, 8}},                             // Google
     {NSAPI_IPv4, {209, 244, 0, 3}},                         // Level 3
     {NSAPI_IPv4, {84, 200, 69, 80}},                        // DNS.WATCH
-    {NSAPI_IPv6, {0x20,0x01, 0x48,0x60, 0x48,0x60, 0,0,     // Google
-                  0,0, 0,0, 0,0, 0x88,0x88}},
-    {NSAPI_IPv6, {0x20,0x01, 0x16,0x08, 0,0x10, 0,0x25,     // DNS.WATCH
-                  0,0, 0,0, 0x1c,0x04, 0xb1,0x2f}},
+    {
+        NSAPI_IPv6, {
+            0x20, 0x01, 0x48, 0x60, 0x48, 0x60, 0, 0, // Google
+            0, 0, 0, 0, 0, 0, 0x88, 0x88
+        }
+    },
+    {
+        NSAPI_IPv6, {
+            0x20, 0x01, 0x16, 0x08, 0, 0x10, 0, 0x25, // DNS.WATCH
+            0, 0, 0, 0, 0x1c, 0x04, 0xb1, 0x2f
+        }
+    },
```

Comment:
There is just this change that looks  fine but not same as the rest of the lines above.

@SeppoTakalo shall I manually fix it?

Reply:
Uhhh.. I did not spot this..
Astyle is trying to be clever and avoiding oneliners. If you cant somehow prevent these lines to be changed, please do.

Reply:
Fixed in 88a7416 commit